src/renderer/services/midgard/types.ts

```diff
@@ -154,3 +154,12 @@ export type PoolSharesLD = LiveData<Error, PoolShares>
 export type ValidatePoolLD = LiveData<ApiError, boolean>
 
 export type ValidateNodeLD = LiveData<ApiError, boolean>
+
+export type HistoryActionsPage = {
+  total: number
+  actions: HistoryActions
+}
+
+export type HistoryActionsPageRD = RD.RemoteData<ApiError, HistoryActionsPage>
+
+export type HistoryActionsPageLD = LiveData<ApiError, HistoryActionsPage>
```

Comment:
needed for next PR. forgot to remove from this one. pls do not pay attention here